.github/workflows/pr.yml

```diff
@@ -30,7 +30,7 @@ jobs:
             ${{ runner.os }}-node-modules-
 
       - name: Install
-        run: yarn install --frozen-lockfile
+        run: yarn install --immutable
```

Comment:
q: you already know what the q is

Reply:
--frozen-lockfile is deprecated, --immutable is more strict and check node_modules also, shouldn't be a big deal for the CI but I did that to remove a warning from the CI

Reply:
Seems sane to me! And web is indeed using that too.